src/System.Security.Permissions/src/System/Security/CodeAccessPermission.cs

```diff
@@ -0,0 +1,20 @@
+namespace System.Security
+{
+    public abstract partial class CodeAccessPermission : System.Security.IPermission, System.Security.ISecurityEncodable, System.Security.IStackWalk
+    {
+        protected CodeAccessPermission() { }
+        public void Assert() { }
+        public abstract System.Security.IPermission Copy();
+        public void Demand() { }
+        public void Deny() { }
+        public override bool Equals(object obj) { return default(bool); }
+        //    public abstract void FromXml(System.Security.SecurityElement elem);
+        public override int GetHashCode() { return default(int); }
```

Comment:
Yeah, I realized that later and added a comment somewhere else.  I'm fine with doing ```public override bool Equals(object obj) => base.Equals(obj);```.  The primary thing I was commenting on was changing the behavior away from what the base type provides.